.github/workflows/opensearch-observability-test-and-build-workflow.yml

```diff
@@ -4,13 +4,18 @@ on: [pull_request, push]
 
 jobs:
   build:
+    env:
+      BUILD_ARGS: ${{ matrix.os_build_args }}
     strategy:
       matrix:
-        java:
-          - 11
-          - 17
-
-    runs-on: ubuntu-latest
+        java: [11, 17]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+          - os: windows-latest
+            os_build_args: -x integTest -x jacocoTestReport
+          - os: macos-latest
+            os_build_args: -x integTest -x jacocoTestReport
```

Comment:
Are integ tests failing? Please open an issue to get those to work on those OSs as well in the future.

Reply:
Done #1159